avm/utilities/pipelines/platform/Invoke-AvmJsonModuleIndexGeneration.ps1

```diff
@@ -0,0 +1,169 @@
+<#
+.SYNOPSIS
+Creates the moduleIndex.json file for the AVM modules that is used by Visual Studio Code and other IDEs to provide the intellisense list of modules from the Bicep public registry.
+
```

Comment:
Missing synopsis for parameters. Usualy not a biggy but to lead by example ....

Reply:
Fair, just forgot